sql/core/src/test/scala/org/apache/spark/sql/CollationSuite.scala

```diff
@@ -1624,4 +1624,22 @@ class CollationSuite extends DatasourceV2SQLBase with AdaptiveSparkPlanHelper {
       }
     }
   }
+
+  test("show collations") {
```

Comment:
please add test for just "SHOW COLLATIONS"

Reply:
Updated
```scala
val df = sql("SHOW COLLATIONS").limit(10)
    checkAnswer(df,
      Seq(Row("SYSTEM", "BUILTIN", "UTF8_BINARY", null, null,
        "ACCENT_SENSITIVE", "CASE_SENSITIVE", "NO_PAD", null),
      Row("SYSTEM", "BUILTIN", "UTF8_LCASE", null, null,
        "ACCENT_SENSITIVE", "CASE_INSENSITIVE", "NO_PAD", null),
      Row("SYSTEM", "BUILTIN", "UNICODE", "", "",
        "ACCENT_SENSITIVE", "CASE_SENSITIVE", "NO_PAD", "75.1.0.0"),
      Row("SYSTEM", "BUILTIN", "UNICODE_AI", "", "",
        "ACCENT_SENSITIVE", "CASE_INSENSITIVE", "NO_PAD", "75.1.0.0"),
      Row("SYSTEM", "BUILTIN", "UNICODE_CI", "", "",
        "ACCENT_INSENSITIVE", "CASE_SENSITIVE", "NO_PAD", "75.1.0.0"),
     Row("SYSTEM", "BUILTIN", "UNICODE_CI_AI", "", "",
       "ACCENT_INSENSITIVE", "CASE_INSENSITIVE", "NO_PAD", "75.1.0.0"),
     Row("SYSTEM", "BUILTIN", "af", "Afrikaans", "",
       "ACCENT_SENSITIVE", "CASE_SENSITIVE", "NO_PAD", "75.1.0.0"),
     Row("SYSTEM", "BUILTIN", "af_AI", "Afrikaans", "",
       "ACCENT_SENSITIVE", "CASE_INSENSITIVE", "NO_PAD", "75.1.0.0"),
     Row("SYSTEM", "BUILTIN", "af_CI", "Afrikaans", "",
       "ACCENT_INSENSITIVE", "CASE_SENSITIVE", "NO_PAD", "75.1.0.0"),
     Row("SYSTEM", "BUILTIN", "af_CI_AI", "Afrikaans", "",
        "ACCENT_INSENSITIVE", "CASE_INSENSITIVE", "NO_PAD", "75.1.0.0")))

```